egs/wsj/s5/steps/cleanup/decode_segmentation.sh

```diff
@@ -62,7 +63,7 @@ srcdir=`dirname $dir`; # The model directory is one level up from decoding direc
 sdata=$data/split$nj;
 
 mkdir -p $dir/log
-[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
+[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh --per-utt $data $nj || exit 1;
```

Comment:
You won't be able to rely on this working correctly until the issue in
https://github.com/kaldi-asr/kaldi/issues/778
is addressed.  In that case the split-data dir would be named differently.